### PR TITLE
[GO-41] Refactor config

### PR DESCRIPTION
**Pull request by im-kulikov**
- db / redis helpers
- config refactoring
- testdb from `config`
- fix / refactor code
- bump vendor

### CODE REVIEW

File: cmd/yaga/project_example/misc/vars.go [hunk 1]
```diff
@@ -1,8 +1,9 @@
 package misc
 
 import (
-	"os"
 	"time"
+
+	"github.com/cryptopay-dev/yaga/config"
 )
 
 const (
```
Comment (jenchik) [inline cmd/yaga/project_example/misc/vars.go:9]: need?
```Config = "config.yml"```

File: cacher/redis/redis_test.go [hunk 2]
```diff
@@ -2,28 +2,29 @@ package redis
...
 	"github.com/cryptopay-dev/yaga/cacher"
+	"github.com/cryptopay-dev/yaga/config"
 	"github.com/go-redis/redis"
 	"github.com/stretchr/testify/assert"
 )
 
 const keyTpl = "test:key:%d"
 
 func closedCacher() cacher.Cacher {
+
```
Comment (pcherednichenko) [inline cacher/redis/redis_test.go]: why add here empty line?

File: cmd/yaga/project_example/misc/vars.go (hunk 1 above)
Comment (im-kulikov) [reply cmd/yaga/project_example/misc/vars.go:9]: fixed

File: cacher/redis/redis_test.go (hunk 2 above)
Comment (im-kulikov) [reply cacher/redis/redis_test.go]: fixed

**Review by jenchik — APPROVED** [review]

**Review by pcherednichenko — APPROVED** [review]

**Pull request merged**